packages/database/test/database.test.ts

```diff
@@ -76,6 +76,20 @@ describe('Database Tests', function() {
     expect(db.ref().toString()).to.equal('https://localhost/');
   });
 
+  it('Can read ns query param', function() {
+    var db = defaultApp.database('http://localhost:80?ns=foo');
+    expect(db).to.be.ok;
+    expect(db.repo_.repoInfo_.namespace).to.equal('foo');
+    expect(db.ref().toString()).to.equal('http://localhost:80/');
```

Comment:
Don't you need to send this namespace to the backend as well?

Reply:
it is sent to the database as a query param in the ws connection. It's not visible in `db.ref()`, but the ws connection looks like
`ws://s-local.fblocal.com:9000/.ws?v=5&ns=foo`